packages/lesswrong/components/ea-forum/EAButton.tsx

```diff
@@ -31,7 +40,7 @@ const styles = (theme: ThemeType): JssStyles => ({
  * Button component with the standard EA Forum styling
  * (see login and sign up site header buttons for example)
  */
-const EAButton = ({style, className, children, classes, ...buttonProps}: {
+const EAButton = ({style, variant="contained", className, children, classes, ...buttonProps}: {
```

Comment:
Love to see this component getting worked on. I know it just got made and I was excited about that too. Still excited.